pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -176,6 +180,9 @@ def _check_inventory(self, lookup_ids=[]):
                             pokemon.get('individual_stamina', 0)
                         ]})
                     matched_pokemon.append(pokemon)
+                    #remove as egg and add as pokemon
+                    inventory.pokemons().remove(pokemon['id'])
+                    inventory.pokemons().add(inventory.Pokemon(pokemon))
```

Comment:
Not sure about that, because even when incubating, it is still an egg and is in the inventory. It never leaves it, it just switches from egg to actual pokemon.